src/Protocol/ActivityPub/Processor.php

```diff
@@ -1343,9 +1351,11 @@ public static function storeReceivers(int $uriid, array $receivers)
 		}
 	}
 
-	private static function storeCapabilities(int $uriid, array $capabilities)
+	private static function storeCapabilities(int $uriid, array $capabilities): array
 	{
+		$restrictions = [];
 		foreach (['pixelfed:canAnnounce' => Tag::CAN_ANNOUNCE, 'pixelfed:canLike' => Tag::CAN_LIKE, 'pixelfed:canReply' => Tag::CAN_REPLY] as $element => $type) {
+			$restricted = true;
```

Comment:
I don't understand why this flag is set to `true` by default?

Reply:
When there are no or empty restrictions, then that action is fully restricted. So we set it to restricted by default and unset it, when it isn't.